lib/cli.js

```diff
@@ -47,6 +48,32 @@ const mkdir = promisify(fs.mkdir);
 const stat = promisify(fs.stat);
 const writeFile = promisify(fs.writeFile);
 
+/**
+ * Loads plugins with the specified names.
+ * @param {{ "import": (name: string) => Promise<any> }} importer An object with an `import` method called once for each plugin.
+ * @param {string[]} pluginNames The names of the plugins to be loaded, with or without the "eslint-plugin-" prefix.
```

Comment:
It's fine to allow both, but we start recommending using the full package name. In the long run, eslintrc will go away and plugin names are not enforced to be `eslint-plugin-xxx`.